numba/misc/dummyarray.py

```diff
@@ -382,15 +382,12 @@ def ravel(self, order='C'):
         if order not in 'CFA':
             raise ValueError('order not C|F|A')
 
-        if self.ndim <= 1:
-            return self
-
-        elif (order in 'CA' and self.is_c_contig or
-                          order in 'FA' and self.is_f_contig):
+        if (order in 'CA' and self.is_c_contig
+                or order in 'FA' and self.is_f_contig):
             newshape = (self.size,)
             newstrides = (self.itemsize,)
             arr = self.from_desc(self.extent.begin, newshape, newstrides,
-                                 self.itemsize)
+                                  self.itemsize)
```

Comment:
This unaligns the indentation (left over from when the previous line said `Array` instead of `self`).
```suggestion
                                 self.itemsize)
```